### ISSUE TITLE
Improve prepare.sh by adding apt install

### ISSUE THREAD

**Issue by asyd** [dask/dask-docker#11]
Hi, 

can you add something like:

```
if [ "$EXTRA_APT_PACKAGES" ]; then
    echo "EXTRA_APT_PACKAGES environment variable found.  Installing."
    apt install -y $EXTRA_APT_PACKAGES
fi
```

Thanks

**Comment by danielfrg** [comment]
Sounds like a good idea and easy to add and test. Could you make a PR for this?

**Comment by danielfrg** [comment]
Fixed on #12
